Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:2.7
+
+ADD . /opt/ashlar
+WORKDIR /opt/ashlar
+
+RUN apt-get update
+RUN apt-get -y autoremove
+RUN apt-get install -y libgeos-dev binutils libproj-dev gdal-bin
```

Comment:
I was sort of thinking about it the other way, i.e. if the list of packages changes then it won't force the autoremove to re-run. But I'm not sure I know what all the factors are.